### ISSUE TITLE
Agregar la posibilidad de subir y guardar un archivo sgf de la partida

### ISSUE THREAD

**Issue by elsantodel90** [elsantodel90/RAAGo#5]

**Comment by pignacio** [comment]
Esto es un archivo de texto que se puede guardar en un TextField, correcto?
Estaría bueno poner el boton para subir el archivo en lugar de pegar el texto, creo que se puede acomodar.
No quiero meterme en líos de guardarlo en algun lugar fuera de la DB, como generalmente se hace con imágenes, porque en este caso a priori son eso, líos :P

**Comment by elsantodel90** [comment]
Sí, exactamente, supongo que texto en la base está perfecto si no tiene un límite de tamaño. Su uso acá sería simplemente subir / bajar, y ni siquiera se usaría siempre porque muchas veces no se anotan los partidos, así que no necesita estructura ni velocidad ni nada.
